src/app/frontend/common/components/probe/component.ts

```diff
@@ -22,4 +22,23 @@ import {Probe} from '@api/root.api';
 export class ProbeComponent {
   @Input() initialized: boolean;
   @Input() probe: Probe;
+
```

Comment:
Very smart from your side! And I was fighting for hours with CSS. Thank you once more!